elements/rh-footer/RhFooter.ts

```diff
@@ -155,7 +158,7 @@ export class RhFooter extends LitElement {
       <slot name="links"></slot>
       ` : html`
       <pfe-accordion on="dark" color-palette="darkest">${children.map((child, index) => staticHtml`
-        <pfe-accordion-${unsafeStatic(child.type)}>
+        <pfe-accordion-${unsafeStatic(child.type)} part="links-accordion-${unsafeStatic(child.type)}">
```

Comment:
the partname should take a static, as it's an AttributePart

```suggestion
        <pfe-accordion-${unsafeStatic(child.type)} part="links-accordion-${child.type}">
```

FWIW you could also make the partnames a list `part="links accordion ${child.type}`

then

```css
rh-footer::part(accordion header) { ... }
```

Reply:
otherwise, LGTM

Reply:
I do like that approach but there isn't a good `:not()` approach.  So, `rh-footer::part(links accordion):not(header)`.  I suppose that's indicative of needing a better naming convention though.